systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -614,6 +704,18 @@ void testIdentityReplicationPolicy() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Tests offset checkpoint/restore in consumer groups with MM2 active-active mode."),
+        steps = {
+            @Step(value = "Deploy clusters and MM2 in active-active setup.", expected = "Active-active MM2 is ready."),
+            @Step(value = "Send and consume messages across both clusters.", expected = "Clients work on both sides."),
+            @Step(value = "Produce new messages, read part from each cluster.", expected = "Offsets diverge and sync is tested."),
```

Comment:
what doest `read part from each cluster` means?

Reply:
read a portion of the messages for source and target cluster => I re-wrote it to be more explicit..